weave-js/src/components/Panel2/PanelString.tsx

```diff
@@ -143,6 +144,7 @@ export const PanelString: React.FC<PanelStringProps> = props => {
   const inputValue = CGReact.useNodeValue(props.input);
   const compValue = CGReact.useNodeValue(config.diffComparand ?? props.input);
   const loading = inputValue.loading || compValue.loading;
+  const {stringFormat} = useContext(WeaveFormatContext);
```

Comment:
Might add `const {spacing} = stringFormat;` to simplify the below?

Reply:
done